cosi.proto

```diff
@@ -3,7 +3,7 @@ package cosi.v1alpha1;
 
 import "google/protobuf/descriptor.proto";
```

Comment:
If you want to follow the CSI model, `cosi.proto` should be generated not manually created.
Instead `spec.md` is the source of truth and contains both the proto and descriptions. The Makefile has a simple bash script to generate the `.proto` file.

Reply:
done